src/Waste.js

```diff
@@ -0,0 +1,47 @@
+// waste.js
+export class Waste {
+    constructor(scene, x, y, fourContainers, mediumGame) {
+        this.scene = scene;
+        this.x = x;
+        this.y = y;
+
+        // Zoznam obrázkov odpadu
+        if (mediumGame) {
+            this.images = ['box', 'newspaper', 'newspaper_roll', 'paper_cup', 'bottle', 'broken_bottle', 'glass', 'mirror', 'shards', 'buckle', 'can', 'key', 'pot', 'scissors', 'bag', 'bottle2', 'crumpled_bottle', 'cup', 'packing', 'chips', 'cleaner', 'soap', 'toothpaste', 'glass', 'glasses', 'jug', 'fries', 'package', 'toilettePaper', 'yogurt', 'box2', 'eggs', 'stick', 'glass2', 'parfume', 'can2', 'foil', 'fork', 'screw', 'spoon'];
+        } else if(fourContainers){
+            this.images = ['box', 'newspaper', 'newspaper_roll', 'paper_cup', 'broken_bottle', 'shards', 'key', 'bag', 'bottle2', 'crumpled_bottle', 'cleaner', 'glass', 'glasses', 'yogurt', 'box2', 'glass2', 'can2', 'fork', 'screw', 'spoon'];
+        }else {
+            this.images = ['bulb', 'button', 'CD', 'teddy', 'toothbrush', 'box', 'newspaper', 'newspaper_roll', 'paper_cup', 'apple2', 'apple', 'banana', 'beet', 'orange', 'bottle', 'broken_bottle', 'glass', 'mirror', 'shards', 'buckle', 'can', 'key', 'pot', 'scissors', 'bag', 'bottle2', 'crumpled_bottle', 'cup', 'packing', 'chips', 'cleaner', 'soap', 'toothpaste', 'glass', 'glasses', 'jug', 'fries', 'package', 'toilettePaper', 'yogurt', 'candle', 'ceramics', 'diapers', 'shoes', 'tshirt', 'box2', 'eggs', 'stick', 'bread', 'egg', 'flower', 'leaves', 'tea', 'glass2', 'parfume', 'can2', 'foil', 'fork', 'screw', 'spoon'];
+        }
+
+        console.log(this.images.length);
```

Comment:
zabudnuty alebo debug?
Moje odporucanie ktore si nikto nepytal je si tam hodit NOTE alebo TODO.

Reply:
specialne toto je iba taka blbost

Reply:
Mozes vymazat log vypis predtym nez to davame do gitu, avsak mne osobne to zatial nevadi. Bolo by fajn kody cistit hned, ale to viem ze sa lahko hovori. My to cele este budeme debugovat a cistit co najviac.

Reply:
To sa vymaze až sa to spojis Lukášovým. Je to pre moje overovanie